packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -233,6 +237,10 @@ export class MirrorNodeClient {
     handleError(error: any, path: string, effectiveStatusCode: number, method: REQUEST_METHODS, allowedErrorStatuses?: number[], requestId?: string) {
         const requestIdPrefix = formatRequestIdMessage(requestId);
         if (allowedErrorStatuses && allowedErrorStatuses.length) {
+            if (error.message === `timeout of ${parseInt(process.env.MIRROR_NODE_TIMEOUT || '10000')}ms exceeded`) {
```

Comment:
I don't think the placement is right.
Aren't we not making it in here right now?

Would it safer to to it outside of the if
```
        if (effectiveStatusCode === MirrorNodeClientError.ErrorCodes.ECONNABORTED) {
            controller.abort();
        }
```

You could do this at the beginning of the handleError

Reply:
Done.